src/Client.js

```diff
@@ -104,6 +105,14 @@ function Client(options) {
 Client.prototype.query = function(expression, options) {
```

Comment:
non related to this PR, but I didn't know `.query` function receives an `options` argument

Reply:
@marrony Same here! #TIL 🚀